### DISCUSSION TITLE
WHY THIS CODE THROWS 403?

### DISCUSSION THREAD

**Discussion by NSH531** [CrowdStrike/falconpy#704] [Q&A] [answered]
I glad for any help[ on this\

nate
```
url = "https://api.crowdstrike.com/alerts/queries/alerts/v1"

#payload={'client_id': 'id'}
files=[

]
headers = {
  'Content-Type': 'application/json',
  'Authorization': 'Bearer '+Mytoken['body']['access_token']
}

response = requests.request("GET", url, headers=headers, data=payload, files=files)

print(response.text)
```

**Comment by jshcodes** [comment, marked as answer]
Hi @NSH531 -

This code does not make use of FalconPy, so I'm not entirely sure.  Since this is a GET operation, passing a `files` array (especially an empty one) is probably unexpected, and the data payload appears to be malformed.  You might also need to check your API key to make sure it has access to the Alerts scope.

The Alerts Service Class is being worked on this week, so `GetQueriesAlertsV1` should be available sometime next week.

In the interim, you can access this operation using the `override` functionality provided by the Uber Class.

```python
from falconpy import APIHarness
uber = APIHarness(client_id="ID_HERE", client_secret="SECRET_HERE")

result = uber.command("", override="GET,/alerts/queries/alerts/v1")
```
